libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerMixpanel.java

```diff
@@ -1274,6 +1274,22 @@ private AnalyticsTrackerMixpanelInstructionsForStat instructionsForStat(
                 instructions = AnalyticsTrackerMixpanelInstructionsForStat.
                         mixpanelInstructionsForEventName("Media Library - Added Video");
                 break;
+            case MEDIA_UPLOAD_STARTED:
+                instructions = AnalyticsTrackerMixpanelInstructionsForStat.
+                        mixpanelInstructionsForEventName("Media Service - Upload Started");
+                break;
+            case MEDIA_UPLOAD_ERROR:
+                instructions = AnalyticsTrackerMixpanelInstructionsForStat.
+                        mixpanelInstructionsForEventName("Media Service - Response Error");
```

Comment:
Would it be better to use the word "Upload" rather than "Response" here (and in `MEDIA_UPLOAD_SUCCESS`)?